### PR TITLE
Add Rekit package.

### PR DESCRIPTION
**Pull request by supnate**

### CODE REVIEW

**Comment by supnate** [comment]
[Rekit](https://github.com/supnate/rekit) is a toolkit for creating and managing Redux + React + React-router projects. This plugin provides menus for a Rekit project.

File: repository/r.json [hunk 1]
```diff
@@ -725,6 +725,17 @@
 			]
 		},
 		{
+			"name": "Rekit",
+			"details": "https://github.com/supnate/rekit-sublime",
+			"description": "Provide sidebar menu for Rekit command line tools.",
+			"releases": [
+				{
+					"sublime_text": "*",
+					"branch": "master"
```
Comment (wbond) [inline repository/r.json]: This will need to be adjusted per https://packagecontrol.io/docs/submitting_a_package#Step_4

**Comment by supnate** [comment]
Hi @wbond, thank you very much for the reminding. Now I have added a valid tag 'v1.0.0'. Would you please to merge the PR? Thanks!

**Comment by wbond** [comment]
You also need to change that line to `"tags": true` instead of `"branch": "master"`

File: repository/r.json [hunk 2]
```diff
@@ -725,6 +725,17 @@
 			]
 		},
 		{
+			"name": "Rekit",
+			"details": "https://github.com/supnate/rekit-sublime",
+			"description": "Provide sidebar menu for Rekit command line tools.",
+			"releases": [
+				{
+					"sublime_text": "*",
+					"tags": "true"
```
Comment (wbond) [inline repository/r.json]: Just no quotes around `true`.

Comment (supnate) [inline repository/r.json]: Done. 😂😂😂

**Pull request merged**